website/docs/r/pi_volume.html.markdown

```diff
@@ -59,6 +59,13 @@ Review the argument references that you can specify for your resource.
 - `pi_anti_affinity_volumes`- (Optional, String) List of volumes to base volume anti-affinity policy against; required if requesting `anti-affinity` and `pi_anti_affinity_instances` is not provided.
 - `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
 - `pi_replication_enabled` - (Optional, Boolean) Indicates if the volume should be replication enabled or not.
+
+  ~> **Note:** `replication_sites` will be populated automatically if set to true and sites are not specified.
+
+- `pi_replication_sites` - (Optional, List) Specifies sites for volume replication.
+
+  ~> **Note:** `pi_replication_sites` will destroy then create new resource if changed.
+
```

Comment:
```suggestion
- `pi_replication_enabled` - (Optional, Boolean) Indicates if the volume should be replication enabled or not.

  **Note:** `replication_sites` will be populated automatically if set to true and sites are not specified.

- `pi_replication_sites` - (Optional, List) Specifies sites for volume replication.

```
This ` ~>` is used for big announcements and all we want is a note to the user. 
We can have a discussion about `force new` later.